services/metadata/backend/data/db.json

```diff
@@ -1223,6 +1184,527 @@
                     ]
                 }
             ]
+        },
+        {
+            "id": "0838",
+            "name": "Bilddatenbank Geoarchäologie",
+            "description": "Die geoarchäologische Sammlung am IPNA (Integrative Prähistorische und Naturwissenschaftliche Archäologie) der Universität Basel umfasst u.a. Block- und Sedimentproben von natürlichen Böden sowie archäologischen Schichten, ferner auch Gesteinen, Silices, Keramik etc. Diese Sammlung kann dank der „I-GEOARCHive“ Bilddatenbank in Form von Fotografien und Dünnschliffscans (mit entsprechenden Mikroskopfotos) einer breiten Öffentlichkeit und insbesondere der Forschung und Lehre zugänglich gemacht werden. Die frei zugängliche „I-GEOARCHive“ ermöglicht den Nutzer*innen das Beschlagworten, Analysieren, Publizieren und Archivieren geoarchäologischer Proben. Darüber hinaus sind mikromorphologische Befunde mit den Dünnschliffscans und dadurch mit den Blockproben und Profilen verknüpft. Durch diese Vernetzung wird eine multidimensionale Publikation und Archivierung mikromorphologischer Bilddaten und ihr Bezug zum originalen Feldbefund möglich.\n\nE: The geoarchaeological collection of the IPAS (Integrative Prehistory and Archaeological Science) at the University of Basel includes block and sediment samples of soils and archaeological deposits as well as rocks, flints, ceramics etc. The „I-GEOARCHive“ image database of thin section scans (and corresponding microscope photos) makes the collection digitally accessible to the public and in particular to research and teaching. The freely accessible „I-GEOARCHive“ database enables users to keyword, analyse, publish and archive geoarchaeological samples. Furthermore, micromorphological observations are linked to the thin section scans and thus to the block samples and profiles. This cross-linking enables multidimensional publication and archiving of micromorphological image data and their relation to original archaeological features.",
```

Comment:
this makes me wonder: should we really have `description` here, if that's copy-pasted from the metadata anyways? shouldn't it rather be pulled from the metadata directly?

Reply:
Both `name` and `description` are. You're right we should have them from thee, but as for temp solution is easier to have it like that. Imo it doesn't make sense to implement any logic until we will decide on the **final** API and data model shapes (flat/nested, etc.).

Reply:
fair enough.  
in that case, feel free to merge